copier.yaml

```diff
@@ -46,8 +65,9 @@ use_python_black:
 
 use_python_mypy:
     type: bool
-    help: Do you want to use a Python type checker (via mypy)?
-    default: yes
+    when: "{{ preset_name == 'production' }}"
+    help: Do you want to use a Python type checker (via mypy)? Recommended for production projects.
+    default: no

```

Comment:
Is it possible to use expressions for the default too? So it defaults to Yes for production preset and No for starter?

Reply:
possible indeed, refined further, done